HW7/EpicDungeonSim.py

```diff
@@ -0,0 +1,187 @@
+import random
+
+
+class Random_Enemy(object):
+    def __init__(self):
+        self.hp = random.randrange(60, 100, 1)
+
+    def atack(self):
+        return random.randrange(15, 30, 1)
+
+
+class Treasure(object):
+    def __init__(self):
+        global hero
+        self.name = random.choice(['Icarus wings',
+                                   'Gephestus hammer',
+                                   'Zeus lightning'])
+        if self.name == 'Gephestus hammer':
+            self.effect = 'Strenght +1'
+            hero.strength += 1
+            hero.finding_treasures.append(self.name)
+        elif self.name == 'Icarus wings':
+            self.effect = 'Agility +1'
+            hero.agility += 1
+            hero.finding_treasures.append(self.name)
+        elif self.name == 'Zeus lightning':
+            self.effect = 'Intellect +1'
+            hero.intellect += 1
+            hero.finding_treasures.append(self.name)
+
+
+class Hero(object):
+    char_class = ''
+
+    def __init__(self, char_c='Warrior'):
+        self.char_class = char_c
+        if self.char_class == 'Warrior':
+            self.strength = 10
+            self.agility = 7
+            self.intellect = 3
+        elif self.char_class == 'Rogue':
+            self.strength = 5
+            self.agility = 10
+            self.intellect = 5
+        elif self.char_class == 'Mage':
+            self.strength = 4
+            self.agility = 2
+            self.intellect = 14
+        self.hp = self.strength * 15
+        self.mp = self.intellect * 15
+        self.dodge = self.agility * 2
+        self.finding_treasures = []
+
+    def atack(self):
+        if self.char_class == 'Warrior':
+            return self.strength * random.randrange(1, 3, 1)
```

Comment:
the same func for all classes. why do we need this check? it seems you wanted to put agility and intellect but missed it